### PR TITLE
Fix: Update some links from the front page

### LINKED ISSUE #23: Dead Links all Around
There are a lot of dead links on the page. Someone should do a pass through everything and connect components.

Some examples:

image

image

### PR DESCRIPTION
**Pull request by gabrielmocanu**
Fix: Update some links from the front page

Resolve #23

### CODE REVIEW

File: content/en/_index.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ footer:
     - title: Getting Started
       links:
         - title: What is a unikernel?
-          link: /docs/concepts/introduction
+          link: /docs/concepts
```
Comment (razvand) [inline content/en/_index.md:105]: This is fixed here: https://github.com/unikraft/docs/pull/36/files

**Review by razvand — CHANGES_REQUESTED** [review]
One change is already part of another PR: https://github.com/unikraft/docs/pull/36/files

The others are OK.

File: content/en/_index.md (hunk 1 above)
Comment (gabrielmocanu) [reply content/en/_index.md:105]: @nderjung  Can we merge #36 and after this, I can update my PR.

**Comment by gabrielmocanu** [comment]
@razvand  Please check again.

**Review by razvand — APPROVED** [review]
All good. Thanks, @gabrielmocanu .

Reviewed-by: Razvan Deaconescu <razvan.deaconescu@cs.pub.ro>

**Comment by nderjung** [comment]
Thanks!

**Pull request merged**